src/smart-components/role/add-role-permissions/add-role-permissions-summary-content.js

```diff
@@ -38,21 +46,32 @@ const AddRolePermissionSummaryContent = ({ selectedPermissions = [], role = {} }
             </Text>
           </GridItem>
           <GridItem span={9}>
-            <Text component={TextVariants.p}>
+            <TextContent component={TextVariants.p}>
               {selectedPermissions.map((permission, index) => (
                 <Text key={index}> {permission.uuid} </Text>
               ))}
-            </Text>
+            </TextContent>
           </GridItem>
+          {hasCostResources && (
+            <React.Fragment>
+              <GridItem span={3}>
+                <Text className="pf-c-title" component={TextVariants.h6}>
+                  Resource definitions:
+                </Text>
+              </GridItem>
+              <GridItem span={9}>
+                <TextContent component={TextVariants.p}>
+                  {flatten(resourceDefinitions.map((def) => def.resources)).map((resource, index) => (
```

Comment:
I don't think this will work either
```JSX
{
  resourceDefinitions.map(
    ({ resources }) => resources.map((resource, index) => <Text key={index}> {resource} </Text>)
  )
}
```

Reply:
It is working, but your suggestion is nicer

Reply:
The reason why it's working is because you probably tested it on single item resources. If there would be more than 1, they'd all be in single Text component.